portfolio/src/main/java/com/google/sps/data/Video.java

```diff
@@ -71,8 +73,14 @@ public static Video videoFromPlaylistItem(PlaylistItem playlistItem) throws Malf
         String videoId = snippet.getResourceId().getVideoId();
         String videoTitle = snippet.getTitle();
         String videoDescription = snippet.getDescription();
-        String thumbnailUrl = snippet.getThumbnails().getDefault().getUrl();
         boolean isPublic = !playlistItem.getStatus().getPrivacyStatus().equals("private");
+        String thumbnailUrl;
+        if (!isPublic) {
+            // Set url to unavailable thumbnail.
+            thumbnailUrl = "http://i.ytimg.com/vi/D5_RLA5NJAY/hqdefault.jpg";
```

Comment:
Let's define this URL as a variable somewhere. 

UNAVAILABLE_THUMBNAIL_URL = <>.

Reply:
Fixed, added as static variable